### PR TITLE
Add template descriptions to interactive selection

### PR DESCRIPTION
**Pull request by achidlow**
Also:
- add playground template to blessed list
- forward questionary output to sys.stdout

### CODE REVIEW

File: src/algokit/cli/init.py [hunk 1]
```diff
@@ -49,12 +49,22 @@ def __str__(self) -> str:
...
 # this is a function so we can modify the values in unit tests
-def _get_blessed_templates() -> dict[str, TemplateSource]:
+def _get_blessed_templates() -> dict[str, BlessedTemplateSource]:
     return {
-        # NOTE: leaving unpinned for now whilst this under active development, but this would be
-        # a good example of a TemplateSource that should have a commit= specified
-        "beaker": TemplateSource(url="gh:algorandfoundation/algokit-beaker-default-template"),
+        "beaker": BlessedTemplateSource(
+            url="gh:algorandfoundation/algokit-beaker-default-template",
+            description="Official starter template for Beaker applications.",
```
Comment (robdmoore) [inline src/algokit/cli/init.py:62]:
```suggestion
description="Official starter template for production Beaker applications.",
```

File: src/algokit/cli/init.py [hunk 2]
```diff
@@ -49,12 +49,22 @@ def __str__(self) -> str:
...
+        ),
+        "playground": BlessedTemplateSource(
+            url="gh:algorand-devrel/playground",
```
Comment (robdmoore) [inline src/algokit/cli/init.py:65]: This works for now, but we need to shift this to the algorandfoundation repo, but we can do that in a separate PR

File: src/algokit/cli/init.py [hunk 3]
```diff
@@ -315,15 +325,27 @@ def validate(self, document: prompt_toolkit.document.Document) -> None:
...
+            questionary.Choice(
+                title=[("bold", key), ("", description_prefix + tmpl.description)],
+                value=tmpl,
+            )
+            for key, tmpl in _get_blessed_templates().items()
+        ]
+        + [
+            # note: can't use None as default value, that gets ignored
+            questionary.Choice(
+                title=f"<other>{description_prefix}Enter a custom URL - potentially dangerous!", value=object()
```
Comment (robdmoore) [inline src/algokit/cli/init.py:342]: Should we expand on dangerous? e.g. ` (grants execution privileges on your machine during installation)`

**Review by robdmoore — APPROVED** [review]

File: src/algokit/cli/init.py (hunk 3 above)
Comment (achidlow) [reply src/algokit/cli/init.py:342]: We do expand on this at length if they select that option, by printing `_unofficial_template_warning`
